d3a_api_client/redis_market.py

```diff
@@ -0,0 +1,52 @@
+import logging
```

Comment:
As a general comment in this file, you could potentially inherit from RedisDeviceClient and override/replace methods in order to achieve the same result as here and to be able to reuse code at the same time. Despite that, not important. Feel free to skip it

Reply:
I definitely thought about it, but i dont want to expose functions like register and unregister to the RedisMarketClient. But i could override them with pass. What do you think?

Reply:
That would be the way yes, to create methods that have an empty body (only a pass statement) in order to mask the base class implementation. Still, if you feel it is not worth it leave it as is

Reply:
I dont like the idea really, because i would have to skip the super().__init__() , too. 
So, omitting inheriting .